boards/arm/lpcxpresso55s36/lpcxpresso55s36-pinctrl.dtsi

```diff
@@ -42,8 +42,8 @@
 	/* Configures pin routing and optionally pin electrical features. */
 	pinmux_sctimer_default: pinmux_sctimer_default {
 		group0 {
-			pinmux = <SCT0_OUT0_PIO1_4>,
-				<SCT0_OUT2_PIO0_15>;
+		  pinmux = <SCT0_OUT0_PIO1_4>,
+			  <SCT0_OUT2_PIO0_15>;
```

Comment:
nit: use tabs instead of spaces (im actually surprised this passed CI, normally it wants tabs too)

Reply:
I don't know how this slipped by, I can change this right away!